lib/Quantization/Base/Base.cpp

```diff
@@ -309,10 +328,10 @@ TensorQuantizationParams chooseQuantizationParams(float min, float max,
   assert((result.offset == 0 || schema != quantization::Schema::Symmetric) &&
          "Symmetric quantization should be centered on 0");
 
-  // The only valid offsets for symmetric quantization with uint8 support are 0
-  // and -128.
-  assert((result.offset == -128 || result.offset == 0 ||
-          schema != quantization::Schema::SymmetricWithUInt8) &&
+  // The only valid offsets for symmetric quantization with unsigned support are
+  // 0 and qmin.
+  assert((result.offset == qmin || result.offset == 0 ||
+          schema != quantization::Schema::SymmetricWithUnsigned) &&
          "Symmetric quantization should be centered on 0");
```

Comment:
assert message needs to be fixed, it's not aligned to what we actually doing. (comment is correct)

Reply:
Done.